Lib/sqlite3/dbapi2.py

```diff
@@ -25,6 +25,8 @@
 import collections.abc
 
 from _sqlite3 import *
+from _sqlite3 import _deprecated_version
+from warnings import warn
```

Comment:
Again, this import should be moved inside the `if name in deprecated_names` block